### ISSUE TITLE
context toolbars not announced by screenreaders

### ISSUE THREAD

**Issue by eschiebel** [tinymce/tinymce#5109]
**What is the current behavior? Describe the bug**
when a context toolbar appears, its presence is not announced by screenreaders, so the SR user has no idea it's there

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
with a screenreader running, create a table, put the cursor w/in the table, the table context toolbar appears. Nothing tells the user the toolbar exists

**What is the expected behavior?**
the screenreader tells the user the toolbar is present and ctrl-f9 will focus it,
or a callback when the toolbar opens so the application can make the announcement.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
5.0.6
all browsers, but verified with safari/voiceover/macos

**Comment by TheSpyder** [comment]
We should definitely ARIA announce something like `context toolbar available`, similar to what we do with https://textbox.io table and image controls.

I don't think we need to say `ctrl+f9 to focus` every time; we list that under keyboard shortcuts in the help dialog (available via `alt+0`, which should be announced every time the editor is focussed).

**Comment by MarcoZehe** [comment]
I would love to see an announcement implemented. I just tried out TinyMCE 5 on your demo sites, and didn't know the toolbar was available. I used the context menu (Applications key), which worked fine, but sometimes knowing that there is a contextual toolbar would be useful.
